### PR TITLE
Many commits (~12) joined in one + Fix data structure for d3 chart

### LINKED ISSUE #5: Fix view for 'largestNetworks' nor 'busiestNetworks'
## problem
Neither 'largestNetworks' nor 'busiestNetworks' are highlighted at first on page load.
When clicking either tab, nothing changes (data could be identical)

## solution
- [x] Fix component / template to reflect active tab
- [x] Check if data is identical for both views
- [x] If not, fix view
- [x] if yes, change mock data to make it more visible

### PR DESCRIPTION
**Pull request by ellvtr**
Fixes #5 #6 #8 #15 

Current commit detached from this repo:
https://github.com/ellvtr/ellvtr-status-page-old-fork/tree/jonas

### CODE REVIEW

File: angular-app/src/app/components/home/home.component.ts [hunk 1]
```diff
@@ -93,39 +93,36 @@ export class HomeComponent implements OnInit {
...
+    // console.log('initGraphData: persistedData', persistedData);
+    const psuedoNodes = persistedData['nodes'];
+    const pseudoLinks = persistedData['links'];
+    that.nodes = [];
+    that.links = [];
+    // Instantiate real Node instances iso literal object:
+    for (const pseudoNode of psuedoNodes) {
+      const node = new Node(pseudoNode['id']);
+      // const node = new Node(pseudoNode['address']); // Here's where the bug is at: 'id' iso 'address'
+      node.x = Math.floor(Math.random() * 600) + 100;
```
Comment (palango) [inline angular-app/src/app/components/home/home.component.ts]: So positions are random at the moment: do you think we can implement something like https://bl.ocks.org/mbostock/4062045

File: misc/angular-app.misc.md [hunk 2]
```diff
@@ -1,56 +1,62 @@
-## angular-app thoughts
+## Review of angular-app
```
Comment (palango) [inline misc/angular-app.misc.md]: Do we still need this now that we have the issues?

File: misc/history-2nd-merge-from-ellvrt.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
+## Context
```
Comment (palango) [inline misc/history-2nd-merge-from-ellvrt.md]: I think you should put that into the commit, can be just copied as it is to not make more work.

File: mock/data/restructuredData.json.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+nodes with match in link:
```
Comment (palango) [inline mock/data/restructuredData.json.txt]: Still required, you found the bug, right?

File: mock/serv.js [hunk 5]
```diff
@@ -14,15 +14,12 @@ app.get('/info', (req, res) => {
   res.sendFile('network-info.json', { root: __dirname + '/data/' });
 });
 app.listen(port, f=>{ console.log(`app listening on path '/info' on port ${port}`) });
-
+/*
```
Comment (palango) [inline mock/serv.js]: Can we just remove this?

File: misc/angular-app.misc.md (hunk 2 above)
Comment (ellvtr) [reply misc/angular-app.misc.md]: I guess not. I copied relevant parts to issues.

File: mock/data/restructuredData.json.txt (hunk 4 above)
Comment (ellvtr) [reply mock/data/restructuredData.json.txt]: Correct. Not required anymore.

File: angular-app/src/app/components/home/home.component.ts (hunk 1 above)
Comment (ellvtr) [reply angular-app/src/app/components/home/home.component.ts]: Sure. I see a lot of potential for improving the d3 chart. I suggest later - not in this PR.

Comment (ellvtr) [reply angular-app/src/app/components/home/home.component.ts]: See also #11 and #16

File: misc/history-2nd-merge-from-ellvrt.md (hunk 3 above)
Comment (palango) [reply misc/history-2nd-merge-from-ellvrt.md]: I don't think it's a big deal, otherwise you can shorten it. Your choice.

**Comment by ellvtr** [comment]
New commit.

**Pull request merged**